security/Makefile

```diff
@@ -0,0 +1,30 @@
+GOBIN ?= go
```

Comment:
One central Makefile please.

Reply:
All that in another PR...
I do not want to do a wholesale change to all the makefiles in this PR.